modules/parquet/src/parquetjs/reader.ts

```diff
@@ -350,11 +350,13 @@ export class ParquetEnvelopeReader {
   constructor(
     read: (position: number, length: number) => Promise<Buffer>,
     close: () => Promise<void>,
-    fileSize: number
+    fileSize: number,
+    options?: any
   ) {
     this.read = read;
     this.close = close;
     this.fileSize = fileSize;
+    this.defaultDictionarySize = options?.defaultDictionarySize || 10000000;
```

Comment:
Add a constant at the top of the file `const DEFAULT_DICTIONARY_SIZE  = 1e6`

Reply:
Done